src/app/App.tsx

```diff
@@ -3,15 +3,17 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import '../../config/i18n/i18n';
 import { router } from './router';
 import './styles/index.css';
+import Header from '../layout/Header';
 
 const queryClient = new QueryClient();
 
 export const App = () => {
-  return (
-    <QueryClientProvider client={queryClient}>
-      <div className="app">
-        <RouterProvider router={router} />
-      </div>
-    </QueryClientProvider>
-  );
+    return (
+        <QueryClientProvider client={queryClient}>
+            <div className="app">
+                <Header />
```

Comment:
Let's move Header to the RootRoute

Reply:
You can use this - https://github.com/Aleksey-Timoschenko/production-app/blob/main/src/app/router/router.tsx like an example